server/src/main/java/edu/colostate/cs/cs414/chessirecoders/JungleServer/server/Controller.java

```diff
@@ -0,0 +1,122 @@
+package edu.colostate.cs.cs414.chessirecoders.JungleServer.server;
+
+import com.esotericsoftware.kryonet.Connection;
+
+public abstract class Controller implements Runnable {
```

Comment:
Are the classes that implement Controller in charge of delegating flow of responses? A short JavaDoc comment couldn't hurt.

Reply:
Added in my latest push